Automattic/wp-calypso
# Add icon definition to launchpad navigator package

Related to:
* #81652
* #81575 (which this PR replaces)

## Proposed Changes

* This PR moves the definition for the launchpad navigator icon into the `@automattic/launchpad-navigator` package
   - We are using the Gridicons `checkmark-circle` icon for now, but pushing the icon into the package will simplify future changes should we make any

### Screenshot

<img width="291" alt="Screenshot 2023-09-12 at 22 43 57" src="https://github.com/Automattic/wp-calypso/assets/3376401/4f0297a0-7d5f-4545-9ef2-1e58b538dd60">

## Testing Instructions

* Run this branch locally or via Calypso.live
* Navigate to a site where you have a task list in Customer Home
* Add `?flags=launchpad/navigator` to the URL
* Verify that the task list icon in the masterbar shows a checkmark icon in a circle

## Pre-merge Checklist



- [N/A] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [N/A] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [N/A] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [N/A] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?
